A document about our Zulip channel (#19)
anitab-org/documentation: Zulip /README.md

> # Hi :smiley: :wave:

Reviewer: the file should be located in the root folder, while we don't have anything else to put inside this folder. You can name it `zulip.md`

> # Hi :smiley: :wave:

Reviewer: ```suggestion
# Zulip
```

> ## If you’re new here, here’s what you can do to get started with this community:

Reviewer: ```suggestion
## Getting started on Zulip

If you’re new here, here’s what you can do to get started with this community:
```

:heavy_check_mark: [Introduce yourself on #newcomers in the #newcomers > Introductions topic and tell us your motivations for joining so we can get to know you and help you on your journey](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers)

:calendar: [Follow our Open Sessions in Calendar Events topic on #announcements or through our AnitaB.org Open Source calendar (PST:clock1:). These are voice calls you can participate in and get to know a bit more our community and projects](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles)

:eyeglasses: Learn about our [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/code_of_conduct.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/reporting_guidelines.md) to keep this community safe for you and others.

> # [To get Acquainted with Zulip](https://zulipchat.com/help/getting-started-with-zulip)  

Reviewer: you could put this under a subsection called "Learning Resources", not a bullet point

> # Here are the main streams you should be aware of:

Reviewer: ```suggestion
# Streams
```

> 1. Ask your questions  on [questions :question:](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions) or [newcomers :new:](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) streams 

Reviewer: This could also be joined with the general streams such as announcements, celebrate and general

> 2. Join streams of your interest such as 

Reviewer: ```suggestion
### Category based

These are streams based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:
```

    1. [coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding):computer:
    2. [design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design):art:
    3. [documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentatio):notebook_with_decorative_cover:
    4. [outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research):microscope:

>     5. [oppoturnites](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities):stars:

Reviewer: opportunities stream could be joined to the general point 4.

    6. [quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance):heavy_check_mark:

> 3. You can join also project-specific streams, such as: 

Reviewer: ```suggestion
### Project-specific

You can join also project-specific streams, such as: 
```

>     1. [mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system):mortar_board:

Reviewer: for each project you can include the repositories which are related to each stream. For example portal has [anitab-org/portal](https://github.com/anitab-org/portal) while mentorship system has 4 repos (mobiles and backend)

    2. [open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs):confetti_ball:
    3. [portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal):tada:
    4. [STEM-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv):tv:
    5. [vms](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms:):sparkler:
    6. [anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio):octocat:

> 4. You can also check our more general streams:

Reviewer: ```suggestion
### General

You can also check our more general streams:
```

    1. [announcements](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements):microphone:
    2. [celebrate](https://anitab-org.zulipchat.com/#narrow/stream/223068-celebrate):sparkler:
    3. [general](https://anitab-org.zulipchat.com/#narrow/stream/212722-general):newspaper:

> 5. Resources:

Reviewer: you can remove this point 5, since this is not related to Zulip
